cmd/phpcs-server/testdata/wordpress.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<ruleset name="PHPCompatibility-WP">
+    <description>WordPress specific ruleset which checks for PHP cross version compatibility.</description>
+
+    <autoload>./../PHPCSAliases.php</autoload>
```

Comment:
Wouldn't this now be a broken path?

Reply:
it would, the point is not so much what is in the file... just that the file is indeed loaded if it is a `theme` or `plugin` project